extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/HudiCommands.scala

```diff
@@ -145,6 +145,22 @@ object HudiCommands {
     TableCommandSpec(cmd, Seq(tableDesc), SHOW_TBLPROPERTIES)
   }
 
+  val CompactionHoodiePathCommand = {
+    val cmd = "org.apache.spark.sql.hudi.command.CompactionHoodiePathCommand"
+    val uriDesc = UriDesc("path", classOf[StringURIExtractor])
+    TableCommandSpec(
+      cmd,
+      Seq.empty,
+      CREATETABLE,
+      uriDescs = Seq(uriDesc, uriDesc.copy(isInput = true)))
```

Comment:
why 2？

Reply:
> why 2？

Like analyze table command

Reply:
Why read? Is there an output for users when compacting?

Reply:
> Why read? Is there an output for users when compacting?

Compaction is merge MOR table's commit log file to a parquet file, so it will read old file and write a new file.

Reply:
then we don‘t need uriDesc.copy(isInput = true)

Reply:
Ok